docs/features/slash-commands/deep-planning.mdx

```diff
@@ -12,6 +12,20 @@ sidebarTitle: "/deep-planning"
 	/>
 </Frame>
 
+## Demo Video
+
+Watch Deep Planning in action as Cline investigates a codebase, asks clarifying questions, and generates a comprehensive implementation plan:
+
+<Frame>
+	<video
+		autoPlay
+		loop
```

Comment:
```suggestion
```
Let the user start the video on their own. It's hard to know where it starts and where it ends based on the loop + autoplay